src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+import Display from './Display';
+import Button from './Button';
+
+function Calculator() {
+  return (
+    <div className="calculator-container">
```

Comment:
- [ ] It would be better if the Calculator.js component is class-based as one of the requirements for this milestone. Please fix this minor issue. 

```suggestion
```javascript
class Calculator extends Component {
  render() {
```